eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,22 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
```

Comment:
I was expecting the net7 targeting pack to be reported as a prebuilt.  Do you know why it isn't?

Reply:
Because of this?

https://github.com/dotnet/msbuild/blob/8326396cb46aff95a385316674ce076ddd0941d6/global.json#L6

Reply:
Oh sure, the 7.0 SDK is carrying the targeting pack it needs.  Thanks for pointing that out.